src/SCRIPTS/BF/background.lua

```diff
@@ -40,18 +40,34 @@ local function run_bg()
             -- assuming when sensor value higher than 0 there is an telemetry connection
             -- only send datetime one time after telemetry connection became available
             -- or when connection is restored after e.g. lipo refresh
-            local now = getDateTime()
-            local year = now.year;
 
-            values = {}
-            values[1] = bit32.band(year, 0xFF)
-            year = bit32.rshift(year, 8)
-            values[2] = bit32.band(year, 0xFF)
-            values[3] = now.mon
-            values[4] = now.day
-            values[5] = now.hour
-            values[6] = now.min
-            values[7] = now.sec
+            if API_VERSION < 1.041 then
+                local now = getDateTime()
```

Comment:
The only thing that comes to mind is instead of having the two branches in the if/else expr, we do something like
```
assert(loadScript(SCRIPT_HOME.."/rtc_"..API_VERSION..".lua"))()`
```

which contains the api-version specific section.